src/summariser.py

```diff
@@ -36,15 +66,19 @@ def cohere_summarise(message: str) -> NonStreamedChatResponse:
 
 
 def summarise_webpage(url: str) -> str:
-    # html_text = get_html_from_url(url)
-    cohere_response: NonStreamedChatResponse = cohere_summarise(url)
+    html_text = langchain_webcontent(url)
+    cohere_response: NonStreamedChatResponse = cohere_summarise(html_text)
     text: str = cohere_response.text
     return text
 
 
 if __name__ == "__main__":
-    url = "https://samharris.substack.com/p/october-7?utm_source=substack&publication_id=471923&post_id=145856573&utm_medium=email&utm_content=share&utm_campaign=email-share&triggerShare=true&isFreemail=true&r=1s4h2t&triedRedirect=true"
     url = "https://conversationswithtyler.com/episodes/nassim-nicholas-taleb-and-bryan-caplan/"
+    url = "https://samharris.substack.com/p/october-7?utm_source=substack&publication_id=471923&post_id=145856573&utm_medium=email&utm_content=share&utm_campaign=email-share&triggerShare=true&isFreemail=true&r=1s4h2t&triedRedirect=true"
+    url = "https://www.restorationbulletin.com/p/standing-on-a-slackline-giorgia-meloni"
+    url = "http://www.restorationbulletin.com/p/standing-on-a-slackline-giorgia-meloni"
+    url = "https://www.bitsaboutmoney.com/archive/working-title-insurance/"
+    url = "https://www.complexsystemspodcast.com/episodes/teaching-trading-ricki-heicklen/"
```

Comment:
This shouldn't be here I imagine? Lots of never-used code. 
If you were using these to test the function, I would recommend putting them in a test file.
Alternatively, possibly better approach is keep these in a separate data file, and pass that as an argument to this script.

Reply:
TODO: move into a test file. This was my hacky approach for testing the page (`ipython --pdb src/summariser.py`)

Reply:
made an issue to [remind todo ](https://github.com/tommylees112/whatsappLLM/issues/3)